flow-report/src/sidebar/sidebar.tsx

```diff
@@ -0,0 +1,86 @@
+/**
+ * @license Copyright 2021 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import {FunctionComponent} from 'preact';
+import {useMemo} from 'preact/hooks';
+import {classNames, useCurrentLhr, useFlowResult, useLocale} from '../util';
+import {SidebarFlow} from './flow';
+
+const Separator: FunctionComponent = () => {
+  return <div className="Separator" role="separator"></div>;
+};
+
+export const SidebarSummary: FunctionComponent = () => {
+  const currentLhr = useCurrentLhr();
+  const url = new URL(location.href);
```

Comment:
note to self, we'll want a utility for setting partial changes to the hash soon :)